tools/witx/src/ast.rs

```diff
@@ -209,12 +211,27 @@ impl Type {
 
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
 pub enum BuiltinType {
-    String,
-    Char8,
-    USize,
-    U8,
+    Char,
```

Comment:
Could you add a comment here mentioning that this `char` is a Unicode scalar value?